userspace/libraries/libfilepicker/model/DirectoryListing.h

```diff
@@ -14,7 +14,7 @@ class DirectoryListing : public Widget::TableModel
 {
```

Comment:
Directory listing and FileListing can become a single class like FileSystemModel if we use a filter Callback 🤔

Reply:
Ah yep, then it would definetly feel less duplicate. Will fix this